src/aleph/vm/models.py

```diff
@@ -174,10 +191,17 @@ async def create(self, vm_id: int, tap_interface: Optional[TapInterface] = None)
                 hardware_resources=self.message.resources,
                 tap_interface=tap_interface,
             )
+
+            try:
+                await vm.setup()
+                # Avoid VM start() method because it's only for ephemeral programs,
+                # for persistent and instances we will use SystemD manager
+                await vm.configure()
+            except Exception:
+                await vm.teardown()
+                raise
+
         try:
-            await vm.setup()
-            await vm.start()
-            await vm.configure()
```

Comment:
maybe we could keep this and just check with 
 if not self.persistent: like we do above?

Reply:
For Instances don't make sense, because always are persistent.

Reply:
yeah I mean just as a way to only have one code path but that's just cosmetic